src/sentry/models/integrationfeature.py

```diff
@@ -109,6 +109,13 @@ def feature_str(self):
 
     @property
     def description(self):
+        from sentry.models import DocIntegration, SentryApp
+
         if self.user_description:
             return self.user_description
-        return Feature.description(self.feature, self.sentry_app.name)
+        integration = None
```

Comment:
You don't need this line, the type-checker and pyright are smart enough to see that it gets a value in both branches on the conditional.